lib/new_relic/control.rb

```diff
@@ -8,7 +8,6 @@
 require 'new_relic/language_support'
 require 'new_relic/helper'
 
-require 'singleton'
```

Comment:
Why did you decide to remove 'singleton' from this class? Is it because it was required but not included?

Reply:
Never mind -- Your great commit message clarified this for me: [NewRelic::Control::SecurityInterface](https://github.com/newrelic/newrelic-ruby-agent/pull/2044/commits/2e8db5d6e66584530e461d6f79a3ba032aa39d03)